resources/assets/scripts/app/controllers/album.coffee

```diff
@@ -40,4 +40,18 @@ angular.module('ponyfm').controller "album", [
         if auth.data.isLogged
             playlists.refreshOwned().done (lists) ->
                 $scope.playlists.push list for list in lists
+
+        $scope.getCachedAlbum = (id, format) ->
+            $scope.isEncoding = true
+
+            cachedAlbum.download('albums', id, format).then (response) ->
+                albumUrl = response
```

Comment:
Can we get away with using only `$scope.albumUrl` here instead of two very similar variables?